package-res/resources/web/pentaho/visual/ccc/_axes/AbstractAxis.js

```diff
@@ -21,7 +21,7 @@ define([
   "use strict";
 
   return Base.extend({
-    constructor: function(chart, axisId, gems) {
+    constructor: function(chart, axisId, mais) {
       this.chart = chart;
```

Comment:
I agree that `mais` isn't clear. The fact it also is a portuguese word doesn't help.